dali/python/nvidia/dali/math.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2017-2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+def _arithm_op(*args, **kwargs):
+    import nvidia.dali.ops
+    # Fully circular imports don't work. We need to import _arithm_op late and
+    # replace this trampoline function.
+    setattr(sys.modules[__name__], "_arithm_op", nvidia.dali.ops._arithm_op)
```

Comment:
I find the following simpler - but it's a matter of taste, I guess.
```suggestion
    global _arithm_op
    _arithm_op = nvidia.dali.ops._arithm_op
```

Reply:
I just copied your code from `data_node.py` :)